packages/smart-contracts/test/contracts/ERC20SwapToPay.test.ts

```diff
@@ -300,4 +343,14 @@ describe('contract: SwapToPay', () => {
       expect(finalContractRequestBalance.toNumber()).to.equals(0);
     });
   });
+  describe('OnlyWhitelisted admin', () => {
```

Comment:
```suggestion
  describe('Admin tasks', () => {
```

Also please test that we can change fee address and percentage